docs/run_publish/references.md

```diff
@@ -349,14 +347,12 @@ This displays the current version.
 
 ### -w, --workers
 
-This will move block fetching and processing into a worker. By default, this feature is **disabled**. You can enable it with the `--workers=<number>` flag.
+**Positive Integer (default: `1`)** - This will move block fetching and processing into the specified number of workers. You can increase it with the `--workers=<number>` flag.
```

Comment:
```suggestion
**Positive Integer (default: `0`)** - This will move block fetching and processing into the specified number of workers. You can increase it with the `--workers=<number>` flag.
```